snippet-extractor-output/snippets.yaml

```diff
@@ -27823,14 +27823,52 @@ Word.Shape#body:member:
 
       console.log("New content control properties:", newControl);
     });
+Word.Shape#textFrame:member:
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/word/45-shapes/manage-shapes-text-boxes.yaml
+
+
+    await Word.run(async (context) => {
+      // Gets the text frame of the first text box in the main document.
+      const shape: Word.Shape = context.document.body.shapes.getByTypes([Word.ShapeType.textBox]).getFirstOrNullObject();
+      shape.load("textFrame");
+      await context.sync();
+
+      if (shape.isNullObject) {
+        console.log("No shapes with text boxes found in main document.");
+        return;
+      }
+
+      console.log("Text frame of first text box:", shape.textFrame);
+    });
+Word.Shape#textWrap:member:
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/word/45-shapes/manage-shapes-text-boxes.yaml
+
+
+    await Word.run(async (context) => {
+      // Gets text wrap properties of the first text box in the main document.
+      const shape: Word.Shape = context.document.body.shapes.getByTypes([Word.ShapeType.textBox]).getFirstOrNullObject();
+      shape.load("textWrap");
+      await context.sync();
+
+      if (shape.isNullObject) {
+        console.log("No shapes with text boxes found in main document.");
+        return;
+      }
+
+      console.log("Text wrap properties of first text box:", shape.textWrap);
```

Comment:
Does this work? Since textWrap is a complex Object, I would think that you'd need to wrap it in a call of JSON.stringify, like this: 
```suggestion
      console.log("Text wrap properties of first text box:", JSON.stringify(shape.textWrap));
```
If works as is, then just ignore this comment.

Reply:
@Rick-Kirkham Yes, this works. In the Script Lab console, this is what it looks like:

<img width="435" height="381" alt="Image" src="https://github.com/user-attachments/assets/0fc3f906-8633-446f-b3b9-d4af2cac692e" />